lib/progressrus.rb

```diff
@@ -84,7 +115,7 @@ def complete(now: Time.now)
   end
 
   def flush
-    stores.each { |store| store.flush(scope, id) }
```

Comment:
Dunno how the performance compares, but you could also use `each_value` and avoid discarding `_store_name`. Feel free to ignore this comment :)